package-lock.json

```diff
@@ -1,7634 +1,8 @@
 {
   "name": "csm_web",
   "version": "1.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
It seems like you have an outdated version of node, which changes the lockfile version here; update your node and revert these changes.